libevm/SmartVM.cpp

```diff
@@ -93,18 +83,11 @@ namespace
 
 		~JitWorker()
 		{
-			DEV_GUARDED(x_mutex)
-				m_finished = true;
-			m_cv.notify_one();
+			push(JitTask::createStopSentinel());
 			m_worker.join();
```

Comment:
Do I understand this correctly that the worker keeps on compiling after it reaches the end of the queue and we are waiting here in the destructor for this to finish?